Importer/cdli_corpus.py

```diff
@@ -40,7 +40,7 @@ def __init__(self):
         """
         self.texts = []
 
-    def chunk_text(self, file_lines):     # pylint: disable =no-self-use
+    def __chunk_text__(self, file_lines):     # pylint: disable =no-self-use
```

Comment:
This should be `_chunk_text`. The double underscores are for 'magic' strings and operator overloads.